backend/src/main/java/kr/momo/config/CorsProperties.java

```diff
@@ -0,0 +1,17 @@
+package kr.momo.config;
+
+import lombok.Getter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.bind.ConstructorBinding;
+
+@ConfigurationProperties(prefix = "security")
+@Getter
+public class CorsProperties {
+
+    private final String[] allowOrigins;
+
+    @ConstructorBinding
+    public CorsProperties(String[] allowOrigins) {
+        this.allowOrigins = allowOrigins;
+    }
```

Comment:
스프링 3.0 버전부터 파라미터가 존재하는 생성자가 하나 뿐이면  `@ConstructorBinding`은 필요 없다고 합니다. 
다온이 공유해주셨던 Properties 관련 Docs를 참고하여 학습하던 도중에 알게되었습니다 ㅋㅋ

> Improved @ConstructorBinding Detection
When using constructor bound @ConfigurationProperties the @ConstructorBinding annotation is no longer required if the class has a single parameterized constructor. If you have more than one constructor, you’ll still need to use @ConstructorBinding to tell Spring Boot which one to use.
>
> For most users, this updated logic will allow for simpler @ConfigurationProperties classes. If, however, you have a @ConfigurationProperties and you want to inject beans into the constructor rather than binding it, you’ll now need to add an @Autowired annotation.

https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-3.0.0-M2-Release-Notes#improved-constructorbinding-detection

Reply:
오 서로 주고 받는 공식문서 리뷰 좋은데요? 😄
반영하도록 하겠습니다!